descriptions/ascat.cwl

```diff
@@ -14,27 +14,27 @@ hints:
 baseCommand: [run_ascat.sh]
 
 inputs:
-  - id: tumor_file
+  - id: input_tumor_bam
     type: File
     inputBinding:
-      prefix: "--tumor_file"
+      prefix: "--input_tumor_bam"
```

Comment:
is this expected to be changed right? just checking is not a unintended search and replace results

Reply:
Yes, these variable names match the expected variable names to create metaworkflows runs through magma